bootstrapper/internal/kubernetes/kubernetes.go

```diff
@@ -237,11 +257,21 @@ func (k *KubeWrapper) JoinCluster(ctx context.Context, args *kubeadm.BootstrapTo
 	if err != nil {
 		return fmt.Errorf("encoding kubeadm join configuration as YAML: %w", err)
 	}
-	log.With(slog.String("apiServerEndpoint", args.APIServerEndpoint)).Info("Joining Kubernetes cluster")
-	if err := k.clusterUtil.JoinCluster(ctx, joinConfigYAML, log); err != nil {
+
+	k.log.With(slog.String("apiServerEndpoint", args.APIServerEndpoint)).Info("Joining Kubernetes cluster")
+	if err := k.clusterUtil.JoinCluster(ctx, joinConfigYAML, k.log); err != nil {
 		return fmt.Errorf("joining cluster: %v; %w ", string(joinConfigYAML), err)
 	}
 
+	k.log.Info("Prioritizing etcd I/O")
+	err = k.etcdIOPrioritizer.PrioritizeIO()
+	if errors.Is(err, etcdio.ErrNoEtcdProcess) {
+		k.log.Warn("Skipping etcd I/O prioritization as etcd process is not running. " +
+			"This is expected if this node is a non-control-plane node.")
+	} else if err != nil {
+		return fmt.Errorf("prioritizing etcd I/O: %w", err)
+	}
```

Comment:
```suggestion
	if peerRole == role.ControlPlane {
		k.log.Info("Prioritizing etcd I/O")
		if err := k.etcdIOPrioritizer.PrioritizeIO(); err != nil {
			return fmt.Errorf("prioritizing etcd I/O: %w", err)
		}
	}
```
Instead of checking the error, we can just skip the priotization if we arent running on a control plane node